amr-wind/physics/SyntheticTurbulence.cpp

```diff
@@ -449,15 +448,16 @@ SyntheticTurbulence::SyntheticTurbulence(const CFDSim& sim)
     //
     // x-direction points to flow direction (convert from compass direction to
     // vector)
-    m_turb_grid.tr_mat.xx() = -std::sin(wind_direction);
-    m_turb_grid.tr_mat.xy() = -std::cos(wind_direction);
+    m_turb_grid.tr_mat.xx() = std::cos(theta_t);
+    m_turb_grid.tr_mat.xy() = std::sin(theta_t);
     m_turb_grid.tr_mat.xz() = 0.0;
+    m_turb_grid.tr_mat.yx() = -std::sin(theta_t);
+    m_turb_grid.tr_mat.yy() = std::cos(theta_t);
+    m_turb_grid.tr_mat.yz() = 0.0;
     // z always points upwards (for now...)
     m_turb_grid.tr_mat.zx() = 0.0;
     m_turb_grid.tr_mat.zy() = 0.0;
     m_turb_grid.tr_mat.zz() = 1.0;
-    // y = z .cross. x
-    m_turb_grid.tr_mat.y() = m_turb_grid.tr_mat.z() ^ m_turb_grid.tr_mat.x();
```

Comment:
@gantech See #400 for a fix to catch this error in future.

Reply:
Thanks Shreyas.